config/initializers/decidim_awesome.rb

```diff
@@ -0,0 +1,11 @@
+# Change some variables defaults
+Decidim::DecidimAwesome.configure do |config|
```

Comment:
We don't need this initializer for the moment (it's optional, and we want all features for the moment).
You can remove this file.